cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for the `MovieSerializer`. This will automatically generate the `create` and `update` methods and simplify the code, as it is specifically designed for model instances.